src/main/java/io/github/externschool/planner/repository/AuthorityRepository.java

```diff
@@ -0,0 +1,12 @@
+package io.github.externschool.planner.repository;
+
+import io.github.externschool.planner.entity.Authority;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AuthorityRepository extends CrudRepository<Authority, String> {
+
+    Authority findByName(String name);
```

Comment:
it's code from the future, don't do that. But at the moment it's ok, don't remove it